style.css

```diff
@@ -31,7 +22,44 @@
   background-image: url("./images/background.svg");
   background-size: 100vw;
   background-repeat: no-repeat;
-  overflow: hidden;
+}
+
+.menu-item-style {
+  display: flex;
+  background: #6070ff;
+  mix-blend-mode: multiply;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  color: #fff;
```

Comment:
[OPTIONAL] It would be better to stop the page from scrolling after opening the Menu for a better user experience.

Reply:
Good point! I will work on that.